docs/data/data-grid/list-view/list-view.md

```diff
@@ -16,14 +16,99 @@ Unlike the default grid view, the list view makes no assumptions on how data is
 
 In order to display data in a list view, a `unstable_listColumn` prop must be provided with a `renderCell` function.
 
-{{"demo": "ListView.js", "bg": "inline"}}
+```tsx
+function ListViewCell(params: GridRenderCellParams) {
+  return <>{params.row.id}</>;
+}
+
+const listColDef: GridListColDef = {
+  field: 'listColumn',
+  renderCell: ListViewCell,
+};
+```
```

Comment:
To connect the snippet with the above paragraphs:
```suggestion
```tsx
function ListViewCell(params: GridRenderCellParams) {
  return <>{params.row.id}</>;
}

const listColDef: GridListColDef = {
  field: 'listColumn',
  renderCell: ListViewCell,
};

<DataGridPro
  unstable_listColumn={listColDef}
  unstable_listView={true}
/>
```
```